migrations/README.md

```diff
@@ -1,8 +1,8 @@
 Migration scripts
 =================
 
-This directory contains scripts designed to mass-edit experiments, projects, plots,
-and other resources found in DynamoDB and S3 to ensure consistent and safe deployments.
+This directory contains records of scripts designed to mass-edit experiments, projects, plots,
+and other resources found in DynamoDB and S3 to ensure consistent and safe deployments. They are not meant to be re-run.
```

Comment:
If we ever have to restore a DB backup they might have to run again. I'd remove that comment as DB migration